deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -397,6 +397,28 @@ cephFileSystems:
         # in hyperconverged settings where the volume is mounted on the same node as the osds.
         csi.storage.k8s.io/fstype: ext4
 
+cephFileSystemVolumeSnapshotClass:
+  enabled: true
+  name: ceph-filesystem-vsc
+  isDefault: true
+  deletionPolicy: Delete
+  # see https://github.com/rook/rook/blob/master/Documentation/ceph-csi-snapshot.md#cephfs-snapshots for available configuration
+  parameters:
+    # The secrets contain Ceph admin credentials.
+    csi.storage.k8s.io/snapshotter-secret-name: rook-csi-cephfs-provisioner
+    csi.storage.k8s.io/snapshotter-secret-namespace: rook-ceph
+
+cephBlockPoolsVolumeSnapshotClass:
+  enabled: true
+  name: ceph-block-vsc
+  isDefault: false
+  deletionPolicy: Delete
+  # see https://github.com/rook/rook/blob/master/Documentation/ceph-csi-snapshot.md#rbd-snapshots for available configuration
```

How about referring instead to the generated docs? Same for cephfs above.
```suggestion
  # see https://rook.io/docs/rook/latest/ceph-csi-snapshot.html#rbd-snapshots for available configuration
```